### PR TITLE
Improve/update

### PR DESCRIPTION
**Pull request by sepandasadi**

### CODE REVIEW

**Comment by sepandasadi** [comment]
@isuttell @corescan Please let me know if you are moving forward with this component so I can fix the tests to pass, or delete the repo. Thanks

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-engines:
```
Comment (isuttell) [inline .codeclimate.yml]: Why did you remove codeclimate?

Comment (sepandasadi) [reply .codeclimate.yml]: Wanted to keep all apps setup structure consistent, but I added it back.

**Review by sepandasadi — COMMENTED** [review]
I wanted to keep the app structure consistent with the rest of ship-components. Just added codeclimate back.

**Review by isuttell — APPROVED** [review]
